Automattic/wp-calypso
# composite-checkout: Update checkout button styles and add Button to package

#### Changes proposed in this Pull Request

This PR updates the button styles in the composite checkout to match the new flat buttons in Callypso. In the process, I also removed a duplicate component from Calypso that also appeared in the composite checkout package.

**Before**
![image](https://user-images.githubusercontent.com/6981253/84311836-fd525500-ab31-11ea-8825-4abd9bd7ace8.png)


**After**
![image](https://user-images.githubusercontent.com/6981253/84311820-f3c8ed00-ab31-11ea-9d71-800af7e6888d.png)


#### Testing instructions



* Load up Calypso locally or visit calypso.live
* Add a product to your cart and visit checkout
* If you don't see the new checkout add this flag to your URL: `?flags=composite-checkout-force`
* Click around and get as close to your purchase as possible to confirm that the buttons don't look broken anywhere
